doc/Development.md

```diff
@@ -0,0 +1,30 @@
+# Development
+
+## Installation
+
+After cloning the repository into a directory called `RecordFlux`, `pip` can be used to install the project in editable mode:
+
+```Console
+pip install --user -e "RecordFlux[devel]"
```

Comment:
Same as above - `pip` vs `pip3`?

Reply:
Maybe we should recommend using a virtual environment? In a virtual environment `pip` (and `python`) always correspond to the python version the environment has been created with.

Reply:
Doesn't that just move the problem to another step (virtualenv vs. virtualenv3)? I would prefer to keep it as simple as possible. I think with `pip3` we are on the safe side. In a quick search couldn't find a distro where `pip3` doesn't to work.

Reply:
Fixed in 2562906.